SPECS/meson/meson.spec

```diff
@@ -1,21 +1,28 @@
 Summary:        Extremely fast and user friendly build system
 Name:           meson
 Version:        0.60.2
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://mesonbuild.com/
 Source0:        https://github.com/mesonbuild/meson/releases/download/%{version}/%{name}-%{version}.tar.gz
 BuildRequires:  gcc
 BuildRequires:  gettext
-BuildRequires:  gmock-devel
-BuildRequires:  gtest-devel
 BuildRequires:  ninja-build
 BuildRequires:  python3-devel
 BuildRequires:  python3-libs
 BuildRequires:  python3-setuptools
 BuildRequires:  python3-xml
```

Comment:
Nit: `python3-xml` can be removed as a BR. It has been folded into the `python3-libs` package in 2.0.

Reply:
I did not know that. Thanks :)